projects/components/src/lib/inline-message/inline-message-title/inline-message-title.component.scss

```diff
@@ -0,0 +1,18 @@
+.wrapper {
+    display: inline;
+}
+
+.title {
+    display: inline;
+}
+.button {
```

Comment:
.button is also the default button class from the muzieh css package. Do we use anything from that class here?